src/App.js

```diff
@@ -1,25 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+// src/App.js
+import React from 'react';
+import Login from './components/Login';
+import Home from './components/Home';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => {
+    return (
+       <Router>
+        <Routes>
+          <Route path='/' exact Component={Login} />
+          <Route path='/home' Component={Home} />
```

Comment:
Need to setup a protected route.
Refer the code : https://github.com/venkatasaipunnam/workshop-react/blob/master/src/App.js#L13